app/src/GeoJSONMap.js

```diff
@@ -1,57 +1,42 @@
 import React, { Component } from "react";
-import ReactMapboxGl, { Popup, Layer, Feature, GeoJSONLayer, ScaleControl, ZoomControl } from "react-mapbox-gl";
+import ReactMapboxGl, { Layer, Feature, GeoJSONLayer, ScaleControl, ZoomControl } from "react-mapbox-gl";
 import config from "./config.json";
 import { getGeoJson } from './MatrixService.js';
-import { Button } from 'react-bootstrap';
 const { accessToken, center } = config;
 
+const MapGL = ReactMapboxGl({
+  accessToken: accessToken
+});
+
 export default class GeoJSONMap extends Component {
   state = {
     center: center,
     markers: new Map(),
     geojson: null,
-    popup: null
+    pitch: [0]
   };
 
+  _markersTolatLngs = function (markers) {
+    return Array.from(markers.values()).map(m => m.latLng)
+  };
 
-  newMarker = function (lngLat) {
+  _newMarker = function (lngLat) {
```

Comment:
What's the point on using underscore convention?
P.S: JS allows encapsulation, if desired. Through the contructor pattern either in [JS](http://www.samselikoff.com/blog/some-Javascript-constructor-patterns/) or [ES](https://medium.com/javascript-scene/javascript-factory-functions-vs-constructor-functions-vs-classes-2f22ceddf33e)

Reply:
For me it marks methods that are not directly called from JSX. Would encapsulation work for that for you?

Reply:
I though you were using the common underscore convention for "internal" methods in Python.